index.bs

```diff
@@ -281,6 +281,33 @@ In these cases, features should be designed such that it is possible for
 code to gracefully fail or be polyfilled.
 See [[#feature-detect]].
 
+<h3 id=new-features>New features can add functionality to the Web</h3>
+
+A new Web feature can add capabilities to the Web with little concern about the effect on existing content.
```

Comment:
```suggestion
New Web features should err on the side of *adding* rather than *changing* or *removing* functionality to avoid breaking existing content.
```

Reply:
This suggestion changes the meaning of the original text in a direction that wasn't intended.